### ISSUE TITLE
Releases area?

### ISSUE THREAD

**Issue by yeltommo** [apastel/ytmusic-deleter#22]

**Comment by apastel** [comment]
Were you able to find the release? Here's a direct link to it: https://github.com/apastel/ytmusic-deleter/releases/download/1.4.0/ytmusic-deleter-1.4.0.exe

However there's a note that I put on that release that you might've seen already: Windows has been flagging it as malicious and does not allow me to run it, at least on my computer. When I looked into it, it seemed like it's a common problem with .exe files built using the tool I used (PyInstaller). If you have the same issue, you will have to use the "Manual Install" instructions for now.
